src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -57,11 +66,15 @@ const actions: ActionTree<IEmptyState, IRootState> = {
     Vue.set(block.config, 'choices', newChoices)
 
     Vue.delete(block.vendor_metadata?.floip.ui_metadata.set_contact_property.property_value_mapping, resourceId)
-    Vue.set(
-      block.config.set_contact_property?.[0] ?? {},
-      'property_value',
-      choicesToExpression(newChoices, block.vendor_metadata?.floip?.ui_metadata?.set_contact_property?.property_value_mapping),
-    )
+
+    const propertyValueMapping = block.vendor_metadata?.floip?.ui_metadata?.set_contact_property?.property_value_mapping
+    if (propertyValueMapping !== undefined) {
```

Comment:
Could we add a comment above this to explain WHY ? It's not obvious at the 1st glance and might takes time to understand this